### ISSUE TITLE
[K4] Missing Panel Tabs in SimpleStats Plugin

### ISSUE THREAD

**Issue by afbora** [Daandelange/kirby3-simplestats#40]
Link to plugin: https://github.com/Daandelange/kirby3-simplestats

- Kirby Version: 4.0.0-alpha.7
- SimpleStats Version: 0.6.2-beta

Clicking initially the "Simple Stats" in the left menu, it shows "Information" in the top menu
![image](https://github.com/getkirby/kirby/assets/88711468/2f5da88d-d83c-4362-ba52-20976e151f5a)

Clicking "SimpleStats" in the top menu, it changes "Information" to "Page Visits" - but page did not change:
![image](https://github.com/getkirby/kirby/assets/88711468/700ec6ab-e006-41b9-a93b-14fb73c14294)

---------------------------------------
Running Version
- Kirby Version: 3.9.6.1
- SimpleStats Version: 0.6.2-beta

![image](https://github.com/getkirby/kirby/assets/88711468/27fc2136-aa17-49c7-9036-8a6174e595b1)

Author @zeljkovorel from https://github.com/getkirby/kirby/issues/5571

**Comment by Thiousi** [comment]
Hello,
I am testing an upgrade to K4 on a rather old website with simplestats 0.5.0-beta.
The tabs do appear, but clicking on them changes only the url, title, breadcrumb but not the tab contents.
![image](https://github.com/Daandelange/kirby3-simplestats/assets/4822666/4330817e-3a46-4f05-80c4-296f9e63e11f)
![image](https://github.com/Daandelange/kirby3-simplestats/assets/4822666/f7d0c59b-a108-4026-920f-c4ecb117ef82)

**Comment by Daandelange** [comment]
Hi, Thanks for this additional information.

I've been struggling with the `k-tabs` item since forever, when it was still supported and documented. It has been changed multiple times during v3 to finally be removed from the docs. Now it's back again showing an empty docs page.  
I couldn't yet find any info about the `k-tabs` state in v4; the Kirby team said they will improve UI Kit docs for v4, let's wait for that !? I really hope they get this done once and for all in v4.

**Comment by miragecraft** [comment]
Is it possible to change SimpleStats to not use k-tabs in order to support Kirby v4+?

I believe Retour's "tabs" are just links to separate pages.

**Comment by Daandelange** [comment]
Yes, ideally that would be a good solution. Sorry for the delay the k4 update is taking, I'm a bit lacking time atm, I'll have a web dev session somewhen this winter ;) (however, I might jump to kirby 5 directly if k4 is too different). Don't hesitate to submit a PR to get it fixed earlier; it should not be much work, it's only the tabs routing and re-implementation. 
_Meanwhile, tracking should work as expected in kirby 4, so data can continue to be collected. (?)_

**Comment by Daandelange** [comment]
While it might break on Kirby 3/4, this issue has been fixed for k5 in 66b6602da0a0ffc605bf0d345743c4fe2e974fd5
Finally I have re-used the always breaking tabs component, we'll see... ^_^
